core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -1680,13 +1701,14 @@ object ZIO extends ZIO_R_Any {
     final val Fold            = 5
     final val Fork            = 6
     final val Uninterruptible = 7
-    final val Supervise       = 8
+    final val Supervised      = 8
     final val Ensuring        = 9
     final val Descriptor      = 10
     final val Lock            = 11
     final val Yield           = 12
     final val Access          = 13
     final val Provide         = 14
+    final val Fibers          = 15
```

Comment:
The other option is to stick this inside `FiberDescriptor` as a `UIO[List[Fiber[_, _]]]`. Although that might slow down the descriptor ever-so-slightly. It would avoid introducing another operation for what is essentially more descriptor data. /cc @regiskuckaertz

Reply:
Nice! Yeah it seems like a good fit.

Reply:
Yes! That's a great idea.